### PR TITLE
Update RepTreeRoot.java for Issue #573

### PR DESCRIPTION
**Pull request by andreakb**
Addresses issue #573

Created and inserted constants for repeated strings

### CODE REVIEW

File: jhove-apps/src/main/java/edu/harvard/hul/ois/jhove/viewer/RepTreeRoot.java [hunk 1]
```diff
@@ -71,6 +71,26 @@ public RepTreeRoot(RepInfo info, JhoveBase base) {
 		snarfRepInfo();
 	}
 
+	private static final String TEXT_MD_METADTA = "TextMDMetadata";
+	private static final String FORMAT = "Format: ";
+	private static final String VERSION = "Version: ";
+	private static final String BYTE_ORDER = "ByteOrder: "
```
Comment (carlwilson) [inline jhove-apps/src/main/java/edu/harvard/hul/ois/jhove/viewer/RepTreeRoot.java]: This line's lacking a trailing semicolon that breaks the build: https://travis-ci.org/github/openpreserve/jhove/jobs/684731746#L4367

**Review by carlwilson — CHANGES_REQUESTED** [review]
Just the trailing semi-colon so that the code compiles. That aside good work and nice choice of variable names, etc.

File: jhove-apps/src/main/java/edu/harvard/hul/ois/jhove/viewer/RepTreeRoot.java (hunk 1 above)
Comment (andreakb) [reply jhove-apps/src/main/java/edu/harvard/hul/ois/jhove/viewer/RepTreeRoot.java]: Added! Sorry about that!

**Review by carlwilson — APPROVED** [review]

**Comment by MartinSpeller** [comment]
[Replace repeated string literals with constants. #573 - Assigned to deanforsmith](https://trello.com/c/QypaLciF/70-replace-repeated-string-literals-with-constants-573-assigned-to-deanforsmith)

**Pull request merged**